assets/Styles/main.css

```diff
@@ -0,0 +1,904 @@
+    margin-top: -2%;
+  }
+
+  #backgroundDesk .hero-content {
+    width: 75%;
+    padding-left: 20%;
+  }
+
+  #backgroundDesk .hero-content h3 {
+    font-size: 45px;
+    align-self: flex-start;
+  }
+
+  #backgroundDesk .hero-content h2 {
+    align-self: flex-start;
+    font-size: 40px;
+  }
+
+  #sumContainer {
+    width: 70%;
+    border: 2px solid white;
+    font-size: 14px;
+  }
+
+  #actContainer {
+    display: grid;
+    grid-template-columns: 20% 20% 20% 20% 20%;
+    padding: 0 10rem 4rem 10rem;
+  }
+
+  #actContainer li {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    text-align: center;
+    justify-items: center;
+  }
+
+  #featSpeakers {
+    padding: 2rem 4rem;
+  }
+
+  #featContainer {
+    flex-direction: row;
+    flex-wrap: wrap;
+    gap: 10px;
+  }
+
+  .featCard {
+    width: 40vw;
+  }
+
+  .featCard h4 {
+    font-size: 19px;
+  }
+
+  .featCard h5 {
+    font-size: 16px;
+    padding-bottom: 5px;
+  }
+
+  footer p {
+    font-size: 18px;
+  }
+
+  .featCard p {
+    font-size: 15px;
+  }
+
+  #partners {
+    padding: 2rem 5rem;
+  }
+
+  #partContainer {
+    padding: 1rem 2rem;
+    justify-content: center;
+  }
+
+  #partnersAbt {
+    display: none;
+  }
+
+  footer {
+    width: 100%;
+    display: flex;
+    justify-content: center;
+  }
+
+  footer div {
+    width: 25%;
+    padding: 3rem 3rem 2rem 2rem;
+  }
+
+  #menuMobile-abt {
+    display: none;
+  }
+
+  #aboutMain {
+    padding-top: 7rem;
+    background-image:
+      url("/assets/png/Ellipse-black-lines.png"),
+      url("/assets/png/Ellipse-color-pattern.png"),
+      url("/assets/png/elipse-B&W.png"),
+      url("/assets/png/elipse-B&W.png"),
+      url("/assets/png/Ellipse-color-lines.png");
+    background-size: 6%, auto, 9%, 7%, 4%;
+    background-position: 89% 3%, 20% 14%, 98% 16%, 2% 8%, 16% 2%;
+  }
+
+  #aboutMain h1 {
+    font-size: 50px;
+    text-align: center;
+    width: 50%;
+    padding-bottom: 4rem;
+  }
+
+  .title {
+    display: none;
+  }
+
+  #abtSummary {
+    width: 55%;
+    margin-bottom: 4rem;
+    padding: 2rem 4rem;
+  }
+
+  .contact {
+    width: 50%;
+    font-size: 16px;
+    padding-bottom: 8rem;
+  }
+
+  #aboutLogo {
+    width: 98%;
+    padding-top: 7rem;
+  }
+
+  #aboutLogo h4 {
+    text-align: center;
+    font-family: var(--secondary-ff);
+    font-weight: 600;
+    font-size: 30px;
+    margin-bottom: 0;
+  }
+
+  .footerAbt p {
+    color: var(--primary-white);
+  }
+
+  #pastEvents p {
+    font-size: 14px;
+    padding-bottom: 5rem;
+  }
+
+  .footerAbt {
+    width: 100%;
+    background-color: var(--primary-black);
+    margin: 0;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+  }
+
+  #aboutLogo p {
+    width: 50%;
+  }
+
+  #pastEvents {
+    padding-bottom: 7rem;
+  }
+
+  #pastEvents h5 {
+    font-size: 30px;
+  }
+
+  #pastEvents div {
+    display: flex;
+    align-items: center;
+    justify-items: center;
```

Comment:
- [ ] **The `justify-items` property doesn't work on flexbox. It is ignored when the container is flex. Please use the appropriate property.**

Please refer to [this MDN guide](https://developer.mozilla.org/en-US/docs/Web/CSS/justify-items) to understand more about `justify-item` property.

Reply:
Fixed :heavy_check_mark: